garage/src/components/SignManifestoModal.tsx

```diff
@@ -0,0 +1,124 @@
+                understand that the power to shape the future lies in our hands.
+                We pledge to use this power responsibly, to construct a virtual
+                space that reflects our shared values and ambitions. We are
+                dedicated to cultivating a sustainable, inventive, and inspiring
+                ecosystem within Tableland and beyond.
+              </ListItem>
+              <ListItem>
+                <span style={{ fontWeight: "bold" }}>We are Community:</span> We
+                are a collective of diverse and unique individuals, bound
+                together by a common vision of a new Internet. We value each
+                member of our community and nurture an environment of
+                inclusivity, respect, and mutual support.
+              </ListItem>
+            </OrderedList>
+
+            <Heading>Your Role</Heading>
+            <Text>
+              As a contributor to the Tableland Mission Board, you are a key
+              player in this pioneering journey. You will have the opportunity
+              to access exclusive opportunities, contribute to the development
+              of the Tableland ecosystem, be rewarded for your efforts in Flight
+              Time (FT) and other perks, as well as having a voice and the right
+              to vote in community decisions.
+            </Text>
+
+            <Heading>Code of Conduct</Heading>
+            <Text>By signing this manifesto, you agree to:</Text>
+
+            <OrderedList listStylePos="inside" spacing={4}>
+              <ListItem>
+                <span style={{ fontWeight: "bold" }}>Respect Others:</span>{" "}
+                Treat all members of the community with respect and kindness.
+                Discrimination, harassment, or any form of disrespectful
+                behavior will not be tolerated.
+              </ListItem>
+              <ListItem>
+                <span style={{ fontWeight: "bold" }}>Collaborate Openly:</span>{" "}
+                Share your knowledge, learn from others, and work together to
+                achieve common goals. The strength of Tableland lies in the
+                collective intelligence of its community.
+              </ListItem>
+              <ListItem>
+                <span style={{ fontWeight: "bold" }}>Uphold Integrity:</span> Be
+                honest, be transparent, and uphold the highest standards of
+                integrity. Your actions reflect on the entire community.
+              </ListItem>
+            </OrderedList>
+            <Text>
+              By signing this manifesto, you are not just joining a mission
+              board; you are becoming a part of Tableland's story. We look
+              forward to your contributions as we shape the future of this
+              virtual frontier together.
+            </Text>
+          </Flex>
+        </ModalBody>
+        <ModalFooter>
+          <Button onClick={onAgree}>I AGREE</Button>
```

Comment:
when a user agrees to the terms, i don't think that data is stored in a table, correct? it'd be a nice little FT bonus we could track in the `missions` table. if you sign the pledge, you get a little FT bump—the `max_number_of_contributions` could be capped at 3000 (1:1 with the # of Rigs).

Reply:
Nope, we just store it in localStorage/cookies. I agree that it would be nice to store it in tableland but I don't think it makes sense unless we can pay the gas for the txn: feels weird to limit users from being able to see the missions & contribute unless they pay for it (most have already paid to buy/mint a rig already), and some might not have eth on arbitrum so they would have to bridge first as well

Reply:
Fair point. It'd be cool if we supported gasless txs for this, but that obviously adds unneeded complexity.